components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -64,184 +64,9 @@ Type               | May be empty ?    | Allows "*" ?         | May specify (una
-
-HTTP request                           | Description
----------------------------------------|------------
-GET /policies/{**id**}/members         | list the membership
-PUT /policies/{**id**}/members         | replace the membership
-POST /policies/{**id**}/members:add    | add to the membership
-POST /policies/{**id**}/members:remove | remove from the membership
-
-### Example Policy Membership Payload
-
-```json
-{
-  "members": [
-    "team:local:admins",
-    "token:admin-token"
-  ]
-}
-```
-
-### Listing Policy Members
-
-```bash
-curl -sSH "api-token: $TOKEN" -X GET \
-https://{{< example_fqdn "automate" >}}/apis/iam/v2/policies/{policy-id}/members?pretty
-```
-
-### Updating All Members on a Policy
-
-Note the `member-id` must be in the correct [Member Expression]({{< relref "iam-v2-guide.md#member-expressions" >}}) format.
-
-Use this HTTP request to replace the policy's membership:
-
-```bash
-curl -sSH "api-token: $TOKEN" -X PUT \
--d '{"members":["{member-id}","{member-id}","{member-id}"]}' \
-https://{{< example_fqdn "automate" >}}/apis/iam/v2/policies/{policy-id}/members?pretty
-```
-
-### Adding Members to a Policy
-
-Specify new members to add to a policy's membership via this HTTP request.
-Ensure each `member-id` is in the correct [Member Expression]({{< relref "iam-v2-guide.md#member-expressions" >}}) format.
-
-```bash
-curl -sSH "api-token: $TOKEN" -X POST \
--d '{"members":["{member-id}","{member-id}"]}' \
-https://{{< example_fqdn "automate" >}}/apis/iam/v2/policies/{policy-id}/members:add?pretty
-```
-
-### Removing Members from a Policy
-
-Specify members to remove from a policy's membership via this HTTP request. Ensure each `member-id` is in the correct [Member Expression]({{< relref "iam-v2-guide.md#member-expressions" >}}) format.
-
-The removed members will still exist within Chef Automate, but are no longer associated with this policy.
-
-```bash
-curl -sSH "api-token: $TOKEN" -X POST \
--d '{"members":["{member-id}","{member-id}","{member-id}","{member-id}"]}' \
-https://{{< example_fqdn "automate" >}}/apis/iam/v2/policies/{policy-id}/members:remove?pretty
-```
-
-## Roles
-
-A role encapsulates a list of actions.
-IAM v2 provides several default *Chef-managed* roles, which are essential to the operation of Chef Automate and cannot be altered.
-Roles that you create are *Custom* roles.
-
-HTTP request              | Description
---------------------------|------------
-GET /roles                | list both *Chef-managed* and *Custom* roles
-GET /roles/{**id**}       | get the specified role
-POST /roles               | create a *Custom* role
-PUT /roles/{**id**}       | update a *Custom* role
-DELETE /roles/{**id**}    | delete a *Custom* role
+These endpoints are now documented [here](https://automate.chef.io/docs/api/#tag/Policies).
```

Comment:
no here links please, maybe `API Documentation for Policies` or `Policies API Documentation` for the link text
https://webaim.org/techniques/hypertext/#link_to_link